packages/framework-integration-tests/src/config/config.ts

```diff
@@ -25,6 +25,17 @@ function configureInvocationsHandler(config: BoosterConfig) {
   }
 }
 
+function configureEventHub(config: BoosterConfig) {
+  config.eventStreamConfiguration = {
+    enabled: true,
+    parameters: {
+      streamTopic: 'test',
+      partitionCount: 3,
+      messageRetention: 1,
```

Comment:
Questions (take into account that my knowledge about Event Hub is very limited):

- Is there a way to manage the partitionCount depending on load? I mean, automatically?
- What would happen if the messageRetention is met? Are events never delivered anymore? Is there a fallback mechanism we could apply? 
    - For example (probably a bit crazy), if the messageRetention is met for some event and we can execute a function before deleting the event, we could fallback to the CosmosDB default "event dispacher" to update read models and call event handlers (good thing is that the event is already stored in the CosmosDB event-stream)

Reply:
1. No, the partition count should be defined
2. If the message retention is met events are removed from the event hub. If we need to archive events beyond the allowed retention period, we can have them automatically stored in Azure Storage or Azure Data Lake. AFAIK there is not mechanisms to execute before deleting an event

Reply:
About the retention: Azure allows you to define a cleanup policy. This policy could be `compact` for an infinite retention time. This option is not available in Terraform